dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -74,7 +74,7 @@ protected String parseCommandText(String commandText) {
                     paramName = "";
                 String dataType = rs.getString(2);
                 DbParameterAccessor dbp = createDbParameterAccessor(
-                        paramName,
+                        '"' + paramName + '"',
```

Comment:
> I just added one new test for this. We need to put there escapped character because its column name and we need to escape collumn name, because in information schema is alway unescapped stored. Meaning that if you use unexcapped character is column always stored as lower case and we can escaped it.

Yes, but what I was wondering is if we can do the escaping somewhere else? I guess for `buildInsertCommand` we can easily handle since it's inside environment class; for `Update` - hmm, maybe we can introduce some general ability for case-sensitive escaping (there is similar issue for other database types). I'm not sure if anything else is affected?